maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -70,18 +73,13 @@ class IOSDriver(
 
     private fun ensureXCUITestChannel() {
         logger.info("[Start] Uninstalling xctest ui runner app on ${iosDevice.deviceId}")
-        IOSUiTestRunner.uninstall()
+        xcUiTestDriver.uninstall()
```

Comment:
Do we need any logs in this method like we had before?

Reply:
The logs are still there they are just shifted to the `XcUiTestDriver` class